### PR TITLE
Remove @wordpress/scripts.

### PR DESCRIPTION
**Pull request by techanvil**
## Summary

Addresses issue:

- #3033 

## Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

**Review by tofumatt — CHANGES_REQUESTED** [review]
~As mentioned in a comment: this approach has licensing issues so we can't use it.~

We should highlight the original MPL license of any code we use straight-up from Gutenberg.

Otherwise this looks good 🙂

File: tests/e2e/script.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
+
+/**
+ * This file was originally copied from the @wordpress/scripts package, version 12.0.0:
+ * https://github.com/WordPress/gutenberg/blob/8e06f0d212f89adba9099106497117819adefc5a/packages/scripts/scripts/test-e2e.js
```
[the remark this answers is not in the document]
Comment (techanvil) [reply tests/e2e/script.js:7]: Thanks @tofumatt, that's a good shout. I've made the change as suggested!

File: package.json [hunk 2]
```diff
@@ -27,20 +27,20 @@
 		"test:visualtest": "npm run backstopjs -- --storybook-host=./dist/",
 		"test:visualapprove": "backstop approve --config=tests/backstop/config.js",
 		"test:analyze": "npm run build:production -- --env.ANALYZE",
-		"test:js": "wp-scripts test-unit-js --config=tests/js/jest.config.js",
-		"test:js:watch": "wp-scripts test-unit-js --config=tests/js/jest.config.js --watch",
+		"test:js": "cross-env BABEL_ENV=test NODE_ENV=test jest --config=tests/js/jest.config.js --passWithNoTests",
```
Comment (tofumatt) [inline package.json:30]: ❤️  for using `cross-env`

File: tests/e2e/script.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+/**
+ * External dependencies
+ */
+/* eslint-disable-next-line jest/no-jest-import */
+const jest = require( 'jest' );
+const { sync: spawn } = require( 'cross-spawn' );
+
+// getArgsFromCLI inlined from @wordpress/scripts utils/process.js v12.0.0.
+// https://github.com/WordPress/gutenberg/blob/8e06f0d212f89adba9099106497117819adefc5a/packages/scripts/utils/process.js#L1-L11
+
```
Comment (tofumatt) [inline tests/e2e/script.js:31]: 🔥 
```suggestion
```

File: tests/e2e/script.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+				arg.startsWith( prefix )
+			);
+		} );
+	}
+	return args;
+};
+
+// getArgFromCLI and hasArgInCLI inlined from @wordpress/scripts utils/cli.js v12.0.0.
+// https://github.com/WordPress/gutenberg/blob/8e06f0d212f89adba9099106497117819adefc5a/packages/scripts/utils/cli.js#L13-L22
+
```
Comment (tofumatt) [inline tests/e2e/script.js:46]:
```suggestion
```

File: tests/e2e/script.js [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+			return ! excludePrefixes.some( ( prefix ) =>
+				arg.startsWith( prefix )
+			);
+		} );
+	}
+	return args;
+};
+
+// getArgFromCLI and hasArgInCLI inlined from @wordpress/scripts utils/cli.js v12.0.0.
+// https://github.com/WordPress/gutenberg/blob/8e06f0d212f89adba9099106497117819adefc5a/packages/scripts/utils/cli.js#L13-L22
```
Comment (tofumatt) [inline tests/e2e/script.js:45]: Thanks for including the commit SHA and not using `trunk` so the lines don't move around 👍🏻

File: package.json [hunk 6]
```diff
@@ -27,20 +27,20 @@
...
+		"test:js:watch": "npm run test:js -- --watch",
```
Comment (tofumatt) [inline package.json:31]: Nice 👍🏻

**Review by tofumatt — APPROVED** [review]
Tested locally and works great. Thanks so much 👍🏻

File: package.json (hunk 2 above)
Comment (techanvil) [reply package.json:30]: Yup, it's a good shout! Straight from the IB that one. :)

**Pull request merged**